frontend/src/components/login/RegisterComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import {FormEvent, useState} from "react";
+import {useNavigate} from "react-router-dom";
+import * as React from "react";
+import {RegisterData, registerPost} from "../../service/tastydate-api-service";
+import {Button, TextField } from "@mui/material";
+
+export default function RegisterComponent() {
+
+    const [userName, setUserName] = useState<string>();
+    const [userPassword, setUserPassword] = useState<string>();
+    const [userPasswordVerify, setUserPasswordVerify] = useState<string>();
+
+    const navigate = useNavigate();
+
+    const register = (event: FormEvent<HTMLFormElement>) => {
```

Comment:
FEATURE: I would do some input validation here. eg. check if both passwords match...

Reply:
OK. I assumed that the backend is more suitable for the check. In that case I am going to implement it in the frontend.

Reply:
I would check these things in the frontend so you can later instanly show the user if something is not correct. You can still verify them in the backend aswell. specaly if you have any password requierments